app/styles/components/post/sidebar/sidebar.css

```
@@ -1,23 +1,23 @@
     position: sticky;
     top: 0;
     right: 0;
 }
 
 .post-sidebar {
-    padding-block-end: var(--spacing-xl);
+    padding-block-end: var(--cc-spacing-xl);
     height: 100%;
     display: flex;
     flex-direction: column;
     align-items: flex-start;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "align-items" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)